src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -508,6 +508,18 @@ theorem getElem_ofBool {b : Bool} {h : i < 1}: (ofBool b)[i] = b := by
 @[simp] theorem zero_eq_one_iff (w : Nat) : (0#w = 1#w) ↔ (w = 0) := by
   rw [← one_eq_zero_iff, eq_comm]
 
+/-- A bitvector is equal to 0#w if and only if all bits are `false` -/
+theorem zero_iff_forall {x: BitVec w} :
```

Comment:
```suggestion
theorem zero_iff_eq_false {x: BitVec w} :
```

I would expect the theorem to be calle something like this.

Reply:
renamed